manifests/a/Autodesk/NavisworksManage2023/2023/Autodesk.NavisworksManage2023.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Autodesk.NavisworksManage2023
+PackageVersion: "2023"
+InstallerType: exe
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: Autodesk_Navisworks_Manage_2023_Win_64bit_dlm\setup.exe
+InstallerSwitches:
+  Silent: -q
+Installers:
+- Architecture: x64
+  InstallerUrl: https://dds.autodesk.com/NetSWDLD/2023/NAVMAN/54026284-5924-352E-895D-2DAE5162F3DE/SFX/Autodesk_Navisworks_Manage_2023_Win_64bit_dlm_001_002.sfx.exe
+  InstallerSha256: D9E0F6B7ACEBF545AAFA6E34855F0A46BF374372D9DFF399DBEFB07221F1F77C
+- Architecture: x64
+  InstallerUrl: https://dds.autodesk.com/NetSWDLD/2023/NAVMAN/54026284-5924-352E-895D-2DAE5162F3DE/SFX/Autodesk_Navisworks_Manage_2023_Win_64bit_dlm_002_002.sfx.exe
+  InstallerSha256: 4e267edd2938d6b72ca06a6e1894d739cba74e7ef65c043f0059c83c0d27df85

```

Comment:
You can't have two different `x64` installers unless they have different `InstallerLocale`